src/forms/index_form.php

```diff
@@ -46,12 +46,12 @@
         <table class="table table-light table-bordered table-hover table-responsive">
             <thead>
             <?php if ($User->isAdmin()) : ?>
-                <th>ID</th>
+                <th class="screen_size">ID</th>
```

Comment:
This class name is too ambiguous. You should opt for something that's clearer of what the class is achieving like 'hide-sm' or something along those lines. Using `hide` somewhere in the name is important